pom.xml

```diff
@@ -61,100 +61,107 @@
   </dependencies>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources-filtered</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
```

Comment:
We include the version number reliably into the resulting JAR package using the filtering applied to the `src/main/resources-filtered/project.properties` file, so we don't need to commit the version number anywhere besides the `pom.xml` file.

Reply:
This should be something that the release script ultimately deals with, but if the build system sets it for us, it's great.

Reply:
Yeah, this seemed like the most streamlined approach for the current setup. Will definitely be something to look into more when we start discussing what the future release process might be.

Reply:
Most SDKs have the version hardcoded in a single place (except for PHP, where we had to create that). I think it's fine, and up to the release process to maintain that. As long as the rest of the system relies on the one authoritative source, I don't think there is anything to fix.

For reference, https://github.com/Learnosity/learnosity-sdk-php/blob/master/release.sh#L84-L95 and https://github.com/Learnosity/learnosity-sdk-python/blob/master/release.sh#L90-L101